packages/salesforcedx-vscode-apex/src/constants.ts

```diff
@@ -96,3 +97,6 @@ export const ENABLE_SOBJECT_REFRESH_ON_STARTUP =
   'enable-sobject-refresh-on-startup';
 export const APEX_EXTENSION_NAME = 'salesforcedx-vscode-apex';
 export const LSP_ERR = 'apexLSPError';
+export const OUTPUT_CHANNEL = vscode.window.createOutputChannel(
+  nls.localize('channel_name')
+);
```

Comment:
Main question i have for the PR: do we want to start having the command output to a separate Apex channel now or later? We could wait until we get all the apex commands in here and switch them all at the same time instead of trickling them one by one into the channel.

Reply:
Agreed, we're planning on waiting to move the test:run functionality over to the Apex channel at the end, so I think we could switch the rest of the Apex commands then too

Reply:
I think we should do it now, no need to wait for every command to be ported.